src/app/effects/book.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { EffectsTestingModule, EffectsRunner } from '@ngrx/effects/testing';
+import {TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {BookEffects} from './book';
+import {GoogleBooksService} from '../services/google-books';
+import {Observable} from 'rxjs';
+import {SearchAction, SearchCompleteAction} from '../actions/book';
+import {Book} from '../models/book';
+
```

Comment:
@vespertilian True, however the current version of `atom-typescript` does not use the TypeScript API directly which causes some issues such as ignoring `typeRoots`. There is going to be a large rewrite of `atom-typescript`  which makes it utilise the TypeScript API.

So what I wrote is a temporary solution until the next version of `atom-typescript` is releases.